userspace/libsinsp/sinsp.cpp

```diff
@@ -681,7 +711,10 @@ void sinsp::open_savefile(const std::string& filename, int fd)
 	params.start_offset = 0;
 	params.fbuffer_size = 0;
 	oargs.engine_params = &params;
-	open_common(&oargs);
+
+	// AFAICT this is because tinfo==NULL when calling the callback in scap_read_fdlist
+	struct scap_platform* platform = scap_savefile_alloc_platform(nullptr, nullptr);
```

Comment:
Note:
We had this before:
```
	if(oargs->mode != SCAP_MODE_CAPTURE)
	{
		oargs->proc_callback = ::on_new_entry_from_proc;
		oargs->proc_callback_context = this;
	}
```

Reply:
Yes, I don't know if it's actually required that we don't call the proc callback for savefiles or it was simply easier that way.